app/code/Magento/Ui/view/base/web/js/lib/knockout/bindings/datepicker.js

```diff
@@ -75,26 +75,21 @@ define([
                 observable = config;
             }
 
-            if (_.isEmpty(observable())) {
-                if ($(element).datepicker('getDate')) {
-                    $(element).datepicker('setDate', null);
-                    $(element).blur();
+            require(['moment', 'mage/utils/misc', 'mage/calendar'], function (moment, utils) {
```

Comment:
Looks like, if we move the `_.isEmpty(observable)` check outside the `require`, we can avoid loading `moment` and ``mage/utils/misc` in some cases, right?